opentelemetry-sdk/benches/log_enabled.rs

```diff
@@ -5,8 +5,8 @@
     Total Number of Cores:   14 (10 performance and 4 efficiency)
     | Test                                        | Average time|
     |---------------------------------------------|-------------|
-    | exporter_disabled_concurrent_processor      |  1.1 ns     |
-    | exporter_disabled_simple_processor          |  4.3 ns     |
+    | exporter_disabled_concurrent_processor      |  2.5 ns     |
```

Comment:
@lalitb @utpilla This regression (paid by everyone, irrespective of which processor/exporter) is why we need to try and improve the cost of checking for suppression. @bantonsson showed some approach to make this faster, will explore that/other-ideas-too subsequently.

Reply:
Should or do we have a tracking issue for this as you right mentioned it affects all exporters (which mayn't generate any logs, or else choose to filter the logs using tokio-filter).